ui/.gitignore

```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+/build/
```

Comment:
I think this is due to the old production build. Move it to the below `# production` section?

```
# production
/build/
/app/build
```

Reply:
Fixed